assets/scss/modules/_forms.scss

```diff
@@ -0,0 +1,391 @@
+form {
+  text-align: center;
+  font-family: $dp-font-family-proximanova;
+
+  .rgpd {
+    margin: 30px 0 10px;
+
+    p {
+      text-align: left;
+      font-size: $dp-font-size-p;
+      margin: 5px 0;
+    }
+  }
+
+  .holder {
+        width: 49%;
```

Comment:
49%? i don't like magic numbers, why not 50% and declare a general atomic class named: .width-half { width: 50%; }